tests/xtimer_mutex_lock_timeout/main.c

```diff
@@ -114,6 +172,77 @@ static int cmd_test_xtimer_mutex_lock_timeout_long_locked(int argc,
             puts("error mutex not locked");
         }
     }
+    /* to make the test easier to read */
+    printf("\n");
+
+    return 0;
+}
+
+/**
+ * @brief   shell command to test xtimer_mutex_lock_timeout
+ *
+ * This function will create a new thread with lower prio
+ * than the main thread (this function should be called from
+ * the main thread). The new thread will get a mutex and will
+ * lock it. This function (main thread) messages the other
+ * thread and calls xtimer_mutex_lock_timeout.
```

Comment:
This part of the documentation is not valid anymore. i.e. the messaging before calling xtimer_mutex_lock_timeout was removed.